java/vector/src/main/java/org/apache/arrow/vector/LargeVarBinaryVector.java

```diff
@@ -112,6 +113,20 @@ public byte[] get(int index) {
     return result;
   }
 
+  /**
+   * Read the value at the given position to the given output buffer.
+   * The caller is responsible for checking for nullity first.
+   *
+   * @param index position of element.
+   * @param outputBuffer the buffer to write into.
+   */
+  public void readBytes(int index, ReusableBuffer<?> outputBuffer) {
```

Comment:
Would we allow any reusable buffer here, when the reusable buffers may be specialized to byte vs string type

Reply:
The idea is that users could implement ReusableBuffer on top of other byte-container types such as ByteBuffer or Netty ByteBuf objects.